pkg/distributors/contracts/MultiRewards.sol

```diff
@@ -398,6 +398,11 @@ contract MultiRewards is IMultiRewards, IDistributor, ReentrancyGuard, Temporari
         IERC20 rewardsToken,
         uint256 reward
     ) external override updateReward(pool, address(0)) {
+        require(
+            rewardData[pool][msg.sender][rewardsToken].rewardsDuration != 0,
```

Comment:
This seems a bit indirect, shouldn't we do `_rewarders[pool][rewardsToken].contains(msg.sender)`?

Reply:
Sure.  fwiw I used the `rewardDuration` check as that's what snx/curve use to check the existance of a reward.  `_rewarders` is not in their implementation since they don't allow multiple rewarders...

Reply:
I see what you mean - I was referrencing this line: https://github.com/curvefi/multi-rewards/blob/master/contracts/MultiRewards.sol#L502

Reply:
@nventuro so we're all set now?